src/layout/drawer/nav/item.tsx

```diff
@@ -0,0 +1,34 @@
+import ListItem from '@mui/material/ListItem'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemIcon from '@mui/material/ListItemIcon'
+import ListItemText from '@mui/material/ListItemText'
+import React from 'react'
+import { NavLink } from 'react-router-dom'
+import styled from 'styled-components'
+
+import { ListItemProps } from './types'
+
+const LayoutDrawerNavItem: React.FC<ListItemProps> = ({ name, path, icon }) => {
+  return (
+    <>
+      <ListItem key={name} disablePadding>
+        <ListItemButton key={name} component={StyledNavLink} to={path}>
+          <ListItemIcon>{icon}</ListItemIcon>
+          <ListItemText primary={name} />
+        </ListItemButton>
+      </ListItem>
+    </>
```

Comment:
`<></>` - przeczytaj sobie https://legacy.reactjs.org/docs/fragments.html
`key` - https://legacy.reactjs.org/docs/lists-and-keys.html#keys

```suggestion
  return (
      <ListItem disablePadding>
        <ListItemButton component={StyledNavLink} to={path}>
          <ListItemIcon>{icon}</ListItemIcon>
          <ListItemText primary={name} />
        </ListItemButton>
      </ListItem>
```

Reply:
Z tym <></> rozumiem, faktycznie nie jest to tu potrzebne. Jeśli chodzi o klucze, to myślałem, że bez tego będzie się wyświetlał error w konsoli przeglądarkowej, ale jak teraz patrzę, to faktycznie nie jest tu potrzebny.

Reply:
`key` jest tylko potrzebny wtedy kiedy renderujesz używając `map` i to tylko do pierwszego komponentu, do zagnieżdżonych już nie trzeba tego przekazywać

Reply:
Ok, dzięki za wyjaśnienie